examples/spawn-and-move/src/models.cairo

```diff
@@ -47,6 +47,14 @@ struct Moves {
     last_direction: Direction
 }
 
+#[derive(Copy, Drop, Serde)]
+#[dojo::model]
+struct Skipped {
```

Comment:
Could be better to have a more realistic name that is only skipped in some profile, for instance an example of a Mock that is only used with the `dev` profile?